src/extension/app/store/app.js

```diff
@@ -1088,6 +1090,16 @@ export class AppStore {
     }
   }
 
+  getBYOMSourceUrl() {
+    const { siteStore: { contentSourcePattern, owner, repo }, location: { pathname } } = this;
+    if (!contentSourcePattern || typeof contentSourcePattern !== 'string') return null;
+    const url = contentSourcePattern
+      .replace('{{org}}', owner)
+      .replace('{{repo}}', repo)
```

Comment:
I would consitently switch from `owner/repo` to `org/site` in all public facing API surfaces.
```suggestion
      .replace('{{site}}', repo)
```